Makefile

```diff
@@ -27,6 +27,7 @@ ASSETS_BUILDDIR ?= lib/web/build
 BINDIR ?= /usr/local/bin
 DATADIR ?= /usr/local/share/teleport
 ADDFLAGS ?=
+TESTFLAGS ?= -log
```

Comment:
Maybe call this `EXTRATESTFLAGS` or something? `-cover`, `-json`, etc are test flags, and they are not included here.

Reply:
Yeah, this new variable name is a bit ambiguous. It was chosen to distinguish it from the existing variables that configure tests: `FLAGS` and `ADDFLAGS`. Seems like these other variables could use some refactoring to help clarify their roles as well, which may help in choosing a better name for this new variable.

Existing variable usage:
- `FLAGS` is defined in the _targets_ of `test-api`, `test-go-root`, `test-go`, `integration`, and `integration-root`; and adds `-race` and `-v`.
- `ADDFLAGS` is defined globally, appended to `BUILDFLAGS`, used in `go list` when defining `PACKAGES` in the `test-go-root` target, and used directly in `test-api`, `test-go-root`, `test-go` targets. It is used to enable builds with tags such as `firestore` and `gcs`. The same variable is also defined in the `build.assets` Makefile.

Renaming boils down to how we distinguish between Go test arguments and _our_ test arguments. `ADDFLAGS` seems to define build flags (tags). And `FLAGS` defines Go test arguments albeit locally in each target. This new variable is a test flag, but is ignored by Go.